t/Renard/Curie/Component/ScrollZoom.t

```diff
@@ -13,23 +13,24 @@ no strict;
 
 my $cairo_doc = CurieTestHelper->create_cairo_document;
 
-fun Scroll_Event( (InstanceOf['Renard::Curie::App']) $app) {
+fun Scroll_Event( (InstanceOf['Renard::Curie::App']) $app, (Int) $direction, (Int) $delta) {
 	my $event = Gtk3::Gdk::Event->new('scroll');
 	$event->state( 'control-mask' );
 	$event->direction('down');
 	$app->page_document_component->signal_emit( 'scroll_event' => $event );
 }
 
-subtest 'Check that ctrl+scroll-up zooms into the page' => fun {
+subtest 'Check that ctrl+scroll-up zooms out of the page' => fun {
 	my ( $app, $page_comp ) = CurieTestHelper->create_app_with_document($cairo_doc);
 
 	print STDERR $page_comp->zoom_level . "\n";
 
 	ok($page_comp->zoom_level - 1.0 < .001, 'Start at Zoom Level 1.0' );
 
-	Scroll_Event( $app );
+	Scroll_Event( $app, 0, 0 );
 
-	print STDERR $page_comp->zoom_level . "\n";
+	print STDERR "Zoom Level is: " . $page_comp->zoom_level . "\n";
+	ok($page_comp->zoom_level - .95 < .001, 'Reduce Zoom Level to .95');
```

Comment:
It would be better to only compare the zoom level against the previous zoom level rather than using a specific value.